plugins/source/digitalocean/codegen/resources.go

```diff
@@ -0,0 +1,373 @@
+package main
+
+import (
+	"github.com/aws/aws-sdk-go-v2/service/s3/types"
+	"github.com/cloudquery/cloudquery/plugins/source/digitalocean/codegen/recipes"
+	"github.com/cloudquery/cloudquery/plugins/source/digitalocean/resources/services/spaces"
+	"github.com/cloudquery/plugin-sdk/codegen"
+	"github.com/cloudquery/plugin-sdk/schema"
+	"github.com/digitalocean/godo"
+)
+
+var Resources = []*recipes.Resource{
+	{
+		Service: "account",
+		ExtraColumns: []codegen.ColumnDefinition{
+			{
+				Name:     "uuid",
+				Type:     schema.TypeString,
+				Resolver: `schema.PathResolver("UUID")`,
+				Options:  schema.ColumnCreationOptions{PrimaryKey: true},
+			},
+		},
+		SkipFields: []string{"UUID"},
+	},
+	{
+		Service: "cdn",
+		ExtraColumns: []codegen.ColumnDefinition{
+			{
+				Name:     "id",
+				Type:     schema.TypeString,
+				Resolver: `schema.PathResolver("ID")`,
+				Options:  schema.ColumnCreationOptions{PrimaryKey: true},
+			},
+		},
+		SkipFields: []string{"ID"},
+	},
+	{
+		Service: "billing_history",
+		Struct:  godo.BillingHistoryEntry{},
+		ExtraColumns: []codegen.ColumnDefinition{
+			{
+				Name:     "invoice_id",
+				Type:     schema.TypeString,
+				Resolver: `schema.PathResolver("InvoiceID")`,
+				Options:  schema.ColumnCreationOptions{PrimaryKey: true},
+			},
+		},
+		SkipFields: []string{"InvoiceID"},
+	},
+	{
+		Service:    "monitoring",
+		SubService: "alert_policies",
+		Struct:     godo.AlertPolicy{},
+		ExtraColumns: []codegen.ColumnDefinition{
+			{
+				Name:     "uuid",
+				Type:     schema.TypeString,
+				Resolver: `schema.PathResolver("UUID")`,
+				Options:  schema.ColumnCreationOptions{PrimaryKey: true},
+			},
+		},
+		SkipFields: []string{"UUID"},
+	},
+
+	{
+		Service: "balance",
```

Comment:
Just curious, what happens when `Struct` is not defined, like in this case?

Reply:
it seems that my tricky regex to remove redundant parms was not very right 😅